patches/src/main/kotlin/app/revanced/patches/spotify/layout/theme/CustomThemePatch.kt

```diff
@@ -125,6 +126,38 @@ private val customThemeBytecodePatch = bytecodePatch {
             SETTINGS_HEADER_COLOR_LITERAL,
             backgroundColorSecondary!!
         )
+
+        // Hijacks a util method that removes alpha to replace hardcoded accent colors
+        removeAlphaFingerprint.match(miscUtilsFingerprint.classDef).method.apply {
+            addInstructions(0, """
+                invoke-static { p0, p1 }, $EXTENSION_CLASS_DESCRIPTOR->replaceColor(J)J
+                move-result-wide p0
+            """)
```

Comment:
Should be formatted like this
```suggestion
            addInstructions(
	            0,
	            """
	                invoke-static { p0, p1 }, $EXTENSION_CLASS_DESCRIPTOR->replaceColor(J)J
	                move-result-wide p0
	            """
           )
```